ehr/resources/schemas/dbscripts/sqlserver/ehr-24.000-24.001.sql

```diff
@@ -0,0 +1,17 @@
+CREATE PROCEDURE ehr.handleUpgrade AS
+BEGIN
+    IF NOT EXISTS(SELECT column_name
+            FROM information_schema.columns
+            WHERE table_name='notificationrecipients' and table_schema='ehr' and column_name='lsid')
+BEGIN
+
+ALTER TABLE ehr.notificationrecipients ADD Lsid LsidType null;
+END
+END;
+GO
+
+EXEC ehr.handleUpgrade
+GO
+
+DROP PROCEDURE ehr.handleUpgrade
+    GO
```

Comment:
Similarly, should work fine, so no change required. But FYI can be done without a PROC. See the `compliance-23.001-23.002.sql` scripts for examples on both PG and SS.